packages/components/psammead-visually-hidden-text/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | --------------------- |
```

Comment:
```suggestion
 | ------- | ----------- |
```

This line is causing the markdown to not render correctly, could you fix it while you're here?